dao/src/test/java/se/fortnox/reactivewizard/db/deserializing/DeserializationPerformanceTest.java

```diff
@@ -0,0 +1,1391 @@
+
+        @Override
+        public boolean isAutoIncrement(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public boolean isCaseSensitive(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public boolean isSearchable(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public boolean isCurrency(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public int isNullable(int i) throws SQLException {
+            return 0;
+        }
+
+        @Override
+        public boolean isSigned(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public int getColumnDisplaySize(int i) throws SQLException {
+            return 0;
+        }
+
+        @Override
+        public String getColumnLabel(int i) throws SQLException {
+            return columnNames[i];
+        }
+
+        @Override
+        public String getColumnName(int i) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public String getSchemaName(int i) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public int getPrecision(int i) throws SQLException {
+            return 0;
+        }
+
+        @Override
+        public int getScale(int i) throws SQLException {
+            return 0;
+        }
+
+        @Override
+        public String getTableName(int i) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public String getCatalogName(int i) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public int getColumnType(int i) throws SQLException {
+            return columnTypes[i];
+        }
+
+        @Override
+        public String getColumnTypeName(int i) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public boolean isReadOnly(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public boolean isWritable(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public boolean isDefinitelyWritable(int i) throws SQLException {
+            return false;
+        }
+
+        @Override
+        public String getColumnClassName(int i) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public <T> T unwrap(Class<T> aClass) throws SQLException {
+            return null;
+        }
+
+        @Override
+        public boolean isWrapperFor(Class<?> aClass) throws SQLException {
+            return false;
+        }
+    }
+
+    public static class MockedResultSet implements ResultSet {
+
+        private final int limit;
+        private int count;
+        private ResultSetMetaData metadata;
+        private String[] stringValues = new String[]{
```

Comment:
Won't this test be optimized by the JVM when containing identical values for every entry, therefore being faster here than in real life? I'm thinking something like Faker might be useful.

Reply:
Yes, this test was removed in a later commit. I decided to avoid tests for the performance.